src/lib/Zikula/Bundle/CoreBundle/EventListener/ThemeListener.php

```diff
@@ -95,23 +97,29 @@ public function setDefaultPageAssets(GetResponseEvent $event)
             return;
         }
         $basePath = $event->getRequest()->getBasePath();
+        $jquery = $this->env == 'prod' ? 'jquery.min.js' : 'jquery.js';
```

Comment:
past practice is based on `env` which I also prefer. `dev` environment means a developer is coding and might need to look at the non-compressed code.